fe/src/pages/user/Profile/Edit/ProfileEdit.jsx

```diff
@@ -1,14 +1,21 @@
+/* eslint-disable react/style-prop-object */
 /* eslint-disable react-hooks/exhaustive-deps */
 import React, { Fragment } from "react";
-import { Link } from "react-router-dom";
 import Tab from "react-bootstrap/Tab";
 import Tabs from "react-bootstrap/Tabs";
 import moment from "moment";
-import LoadingSpinner from "../../../../components/LoadingSpinner";
-import Header from "../../../../components/Header";
 import EditUser from "./components/EditUser";
 import CurrentProfile from "../Index/components/CurrentProfile";
 import { useEditProfile } from "../hooks/useEditProfile";
+import Container from "../../../../shared/components/Layout/Container/Container";
+import GridRow from "../../../../shared/components/Layout/Grid/GridRow";
+import GridColumn from "../../../../shared/components/Layout/Grid/GridColumn";
+import Card from "../../../../shared/components/Card/Card";
+import LayoutCenterChildren from "../../../../shared/components/Layout/Positioning/LayoutCenterChildren";
+import Avatar from "../../../../shared/components/Image/Avatar";
+import ButtonNavLink from "../../../../shared/components/Button/ButtonNavLink";
+import LoadingPlainText from "../../../../shared/components/Spinner/LoadingPlainText";
+import LoadingSpinner from "../../../../shared/components/Spinner/LoadingSpinner";
```

Comment:
- Please try to shorten the imports to maybe `@component/buttons/...`
- check [here](https://javascript.plainenglish.io/how-to-shorten-import-paths-in-react-and-typescript-nodejs-5b257ee0c3fa). you may search for other informations.